The message error is not displaying in case of service init timeout

How to reproduce:
* Start a Kuzzle stack without redis
* Kuzzle will init
* Services will init
* The service internalCache will timeout
* The error looks like:
```
[x] [ERROR] Error
at InternalError.KuzzleError (/opt/kuzzle/node_modules/kuzzle-common-objects/lib/errors/kuzzleError.js:17:19)
at new InternalError (/opt/kuzzle/node_modules/kuzzle-common-objects/lib/errors/internalError.js:6:15)
at list.(anonymous function).init.timeout.catch.err (/opt/kuzzle/lib/services/index.js:138:17)
at tryCatcher (/opt/kuzzle/node_modules/bluebird/js/release/util.js:16:23)
at Promise._settlePromiseFromHandler (/opt/kuzzle/node_modules/bluebird/js/release/promise.js:512:31)
at Promise._settlePromise (/opt/kuzzle/node_modules/bluebird/js/release/promise.js:569:18)
at Promise._settlePromise0 (/opt/kuzzle/node_modules/bluebird/js/release/promise.js:614:10)
at Promise._settlePromises (/opt/kuzzle/node_modules/bluebird/js/release/promise.js:689:18)
at Async._drainQueue (/opt/kuzzle/node_modules/bluebird/js/release/async.js:133:16)
at Async._drainQueues (/opt/kuzzle/node_modules/bluebird/js/release/async.js:143:10)
at Immediate.Async.drainQueues (/opt/kuzzle/node_modules/bluebird/js/release/async.js:17:14)
at runCallback (timers.js:637:20)
at tryOnImmediate (timers.js:610:5)
at processImmediate [as _immediateCallback] (timers.js:582:5)
```

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "json-stable-stringify": "1.0.1",
     "json2yaml": "^1.1.0",
     "jsonwebtoken": "^7.4.1",
-    "kuzzle-common-objects": "3.0.4",
+    "kuzzle-common-objects": "3.0.5",
     "lodash": "4.17.4",
     "moment": "2.18.1",
     "ms": "^2.0.0",

